drivechain-server/bdk/bdk.go

```diff
@@ -62,9 +87,14 @@ func (w *Wallet) exec(ctx context.Context, args ...string) ([]byte, error) {
 		return nil, errors.New("bdk: exec: empty command")
 	}
 
-	cmd := exec.CommandContext(ctx, "bdk-cli", fullArgs...)
+	bdkCliPath, err := getBdkCliPath()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get bdk-cli path: %w", err)
+	}
```

Comment:
Would be nice if we could turn this into an operation that only happens once. I.e. the first time we invoke a command. It's fine to leave the temp file around if it's in a temp directory, IMO

Reply:
Done!